### PR TITLE
Table charts redesign

### PR DESCRIPTION
**Pull request by lojmax**
Redesign of chart-table and time-series-chart-table plugins

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/plugins/plugin-chart-table/src/Styles.tsx [hunk 1]
```diff
@@ -27,29 +27,66 @@ export default styled.div`
...
+
+  tr {
+    border: 1px solid #E6E6E7;
   }
 
   thead > tr > th {
     padding-right: 1.4em;
     position: relative;
-    background: ${({ theme: { colors } }) => colors.grayscale.light5};
+    background: ${({theme: {colors}}) => colors.grayscale.light5};
```
Comment (EvilDrW) [inline superset-frontend/plugins/plugin-chart-table/src/Styles.tsx:57]: did you run this through prettier?

File: superset-frontend/plugins/plugin-chart-table/src/Styles.tsx [hunk 2]
```diff
@@ -27,29 +27,66 @@ export default styled.div`
...
+    text-transform: capitalize;
+    font-feature-settings: 'tnum' on, 'lnum' on;
+    color: #6C6C6E;
   }
   th svg {
     color: ${({ theme: { colors } }) => colors.grayscale.light2};
     position: relative;
     vertical-align: middle;
-    margin: 0 ${({ theme: { gridUnit } }) => gridUnit}px;
+    margin: 0 ${({theme: {gridUnit}}) => gridUnit}px;
```
Comment (EvilDrW) [inline superset-frontend/plugins/plugin-chart-table/src/Styles.tsx:79]: same question, was this run through prettier?

File: superset-frontend/plugins/plugin-chart-table/src/Styles.tsx (hunk 1 above)
Comment (EvilDrW) [inline superset-frontend/plugins/plugin-chart-table/src/Styles.tsx:57]: was this run through prettier?

**Review by EvilDrW — APPROVED** [review]
seems fine to me, but i would like to see how it appears compared to the requested design.

File: superset-frontend/plugins/plugin-chart-table/src/Styles.tsx (hunk 2 above)
Comment (lojmax) [reply superset-frontend/plugins/plugin-chart-table/src/Styles.tsx:79]: It wasn't i think IDE edited it. I'll revert it back to the way it was before.

**Pull request merged**